### PR TITLE
Use citations via BibTeX 

### PR DESCRIPTION
**Pull request by RobertPincus**
A first implementation, probably we'd like to polish this. 

Note that `requirements.txt` specifies `sphinxcontrib-bibtex<2.0.0` but this seems to restrict the ability to set the format and might explain why this implementation differs from what's suggested by [the documentation](https://jupyterbook.org/content/citations.html).

### CODE REVIEW

**Comment by RobertPincus** [comment]
@d70-t @tmieslinger I don't know if either of you have time and/or bandwidth to look at this

File: how_to_eurec4a/intro.md [hunk 1]
```diff
@@ -16,7 +16,13 @@ If you miss some information that could be valuable, feel free to check the link
...
+* EUREC4A overview papers:{cite}`Bony:2017` and {cite}`Stevens:2021`
 * [eurec4a GitHub repository](https://github.com/eurec4a)
 * [AERIS data server](https://observations.ipsl.fr/aeris/eurec4a-data/)
 * [AERIS leaflet](https://observations.ipsl.fr/aeris/eurec4a/Leaflet/index.html)(data visualization)
+
+## References
+```{bibliography} references.bib
+:filter: docname in docnames
+:style: alpha
+```
```
Comment (d70-t) [inline how_to_eurec4a/intro.md:28]: I thought, in stead of putting bibliographies at the end of every chapter, we might want to create a separate "References" chapter at the end of the book which contains all of those? - But that's a matter of taste, other opinions are welcome as well.

**Review by d70-t — COMMENTED** [review]
Thanks @RobertPincus for having a take on this! It already looks better than all of my tries.

We've initially restricted `sphinxcontrib-bibtex<2.0.0` as the upgrade to v2 was just shortly before we started the book. At that time, jupyter book was simply not working with the newer `sphinxcontrib-bibtex`. I think they now have sorted these things out and we can likely get rid of the explicit dependency on `sphinxcontrib-bibtex` in `requirements.txt`, but maybe have to require at least a certain version of `jupyter-book`, as they started at some point to pin `sphinxcontrib-bibtex` to a compatible version.

If possible with the new version, I'd prefer the inline author year citation style (i.e. Pincus et al. (2021) in stead of those cryptic single letter abbreviations).

File: how_to_eurec4a/intro.md (hunk 1 above)
Comment (RobertPincus) [reply how_to_eurec4a/intro.md]: All the references are now in a single chapter.

**Comment by RobertPincus** [comment]
@d70-t I would prefer the Author et al., YEAR stye for in-line citations too, but "{cite:ps}`perez2011python`" as suggested in the [documentation](https://jupyterbook.org/content/citations.html) elicits an error message in my local installation
```
how_to_eurec4a/how_to_eurec4a/intro.md:19: WARNING: Unknown interpreted text role "cite:ts".
```
Perhaps we should merge the PR and fix the aesthetics later?

**Comment by d70-t** [comment]
I agree, let's move forward and fiddle with aesthetics later.

**Pull request merged**